rt/databinding/jaxb/src/main/java/org/apache/cxf/jaxb/WrapperHelperClassLoader.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.cxf.jaxb;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+
+import org.apache.cxf.Bus;
+import org.apache.cxf.common.classloader.ClassLoaderUtils;
+import org.apache.cxf.databinding.WrapperHelper;
+
+public class WrapperHelperClassLoader implements WrapperHelperCreator {
+    public WrapperHelperClassLoader() {
+    }
+
+    @Override
+    public WrapperHelper compile(Bus bus, Class<?> wrapperType, Method[] setMethods, Method[] getMethods,
+                                 Method[] jaxbMethods, Field[] fields, Object objectFactory) {
+
+        int count = 1;
+        String newClassName = wrapperType.getName() + "_WrapperTypeHelper" + count;
+
+        Class<?> cls = null;
+        try {
+            cls = ClassLoaderUtils.loadClass(newClassName, WrapperHelperClassLoader.class);
+        } catch (ClassNotFoundException e) {
+            e.printStackTrace();
```

Comment:
Please log the exception here (debug or trace), just to help with troubleshooting (no `e.printStackTrace();` please)